opensearch/src/main/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactory.java

```diff
@@ -191,33 +202,140 @@ private Optional<ExprType> type(String field) {
   }
 
   /**
-   * Only default strict_date_optional_time||epoch_millis is supported,
-   * strict_date_optional_time_nanos||epoch_millis if field is date_nanos.
-   * <a href="https://opensearch.org/docs/latest/opensearch/supported-field-types/date/#formats">
-   *   docs</a>
-   * The customized date_format is not supported.
+   * return the first matching formatter as an Instant to UTF.
```

Comment:
```suggestion
   * Parses value with the first matching formatter as an Instant to UTF.
```

Reply:
Done in https://github.com/Bit-Quill/opensearch-project-sql/pull/268/commits/58a8f193ffe1407665734d55a60e53ff9bc97aa8